src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars_star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars_star:first-child,
```

Comment:
not a big mistake, but it is better to have the same structure and use `nth-child` instead `first-child`